src/Components/Filter/Filter.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { SelectOption } from '../../App';
+import cn from 'classnames';
+
+type Props = {
+  option: string;
+  onSetOption: (v: string) => void;
+};
+
+export const Filter: React.FC<Props> = ({ option, onSetOption }) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      {Object.values(SelectOption).map((filterType, index) => (
+        <a
+          href="#/"
+          key={index}
```

Comment:
It's a bad practice to use index as key
```suggestion
      {Object.values(SelectOption).map((filterType) => (
        <a
          href="#/"
          key={filterType}
```